analyses/infercnv-consensus-cell-type/scripts/utils.R

```diff
@@ -0,0 +1,72 @@
+# This file contains functions consumed by scripts in this module.
+
+#' Prepare and export an internal reference inferCNV annotation file
+#'
+#' @param reference_name Reference name to create, as recorded in the reference_celltype_tsv
+#' @param reference_celltype_tsv TSV with reference names and associated consensus cell types
+#' @param celltype_tsv TSV with per-cell cell types to consider when creating the reference
+#' @param annotation_file Path to export inferCNV annotations file
+#' @param library_id Library id for the SCE being processed
+#' @param testing Logical for whether we are running with test data
+prepare_internal_reference_annotations <- function(
```

Comment:
Since this is Ewing specific, maybe we note that in the function name? 
```suggestion
prepare_internal_reference_annotations_ewing <- function(
```

Reply:
We'd need a wrapper function to put this in to call from from the infercnv script itself, is why I figured to punt this to later! I was planning to explain when re-requesting review once CI finished but you beat my to the review :)

So, I'd prefer to either:
- keep this as is, or
- actually make a real wrapper function

do you have a preference?

Reply:
Ah I see what you're saying... I think I might punt this for later but then make a comment in the main inferCNV script where you call this function and file an issue about a future TODO.

Reply:
I left another TODO in https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/1126/commits/f37c1fb5b4353df2b24185d4312b5c401cc8f7f1, and I will open the issue after this goes in so that I can link the TODO spots we have to come back to